lib/api/patch/v1/mocks.ts

```diff
@@ -0,0 +1,16 @@
+import { Patch, ResponsibleEntity } from "./types";
+
+export const mockResponsibilityEntity: ResponsibleEntity = {
+  id: "5f82d558-f2c6-4b1c-95dd-0422ab2b11cd",
+  name: "Charlie Doe",
+  responsibleType: "HousingOfficer",
+};
+
+export const mockPatch: Patch = {
+  id: "2fa90983-94b7-4270-a485-dc42ede5af17",
+  parentId: "bcf2cabe-bccc-4bfd-a10c-2fc6b58e9782",
+  name: "SN4",
+  patchType: "patch",
+  domain: "Housing Management",
+  responsibleEntities: [mockResponsibilityEntity],
+};
```

Comment:
How do these changes tie into the `Asset` and `AssetCharacteristics` model changes?

Reply:
I figured for completeness I would mock all the models we have that asset depends on and build up the asset based on those so that this solution is more re-usable and we can easily access mocks for any of these objects in future.
Otherwise I'd just be hard-coding an asset object in isolation and it could only be used specifically for tests with the full asset model.

Reply:
For these two specifically, they're used in the asset model under asset.patches, so they're imported and used in the asset mock.